examples/hello_world.go

```diff
@@ -75,26 +75,20 @@ func Secret(e webui.Event) any {
 }
 
 func Check(e webui.Event) any {
-
-	// Create new JavaScript object
-	js := webui.NewJavaScript()
-
-	// Run the script
-	if !e.Window.Script(&js, "return document.getElementById('MyInput').value;") {
-
-		// There is an error in our script
-		fmt.Printf("JavaScript Error: %s\n", js.Response)
-
+	// Run script
+	resp, err := e.Window.Script("return document.getElementById('MyInput').value;", webui.JavaScript{})
```

Comment:
Maybe this one should be `webui.JavaScript{BufferSize: 1024}` because we expect a response

Reply:
In that case the default Buffersize of 8*1024 would be used. But 1024 is sufficient. Good change 👍